### ISSUE TITLE
Electric Pump

### ISSUE THREAD

**Issue by ut-64bit** [mrh0/createaddition#671]

**Comment by mrh0** [comment]
i have thought of adding this, not sure though

**Comment by ut-64bit** [comment]
Don't know the details of the block?
Or don't know how to make one?
If you need more information about block, I can help you.

**Comment by mrh0** [comment]
no, its just that i goes against the principles i set of this mod, one of which is to not make electric versions of existing blocks in create

**Comment by Dorpi43** [comment]
Just connect electric motor to regular pump. What a problem...
